tests/nataliia_romanchuk/test_key_words.py

```diff
@@ -0,0 +1,106 @@
+def test_not():
+    assert not (1 == 11)
+    assert not (1 == '11')
+    assert not (1 == '1')
+    assert not (11 == '1' + '1')
+    assert not (11 == '10' + '1')
+    assert not (1 == None)
+    assert not (1 == False)
+
+
+def test_and():
+    assert (1 == 1) and ('1' == '1')
+    assert not ((1 == 1) and ('1' == 1))
+
+
+def test_or():
+    assert (1 == 1) or (1 == '1')
+    assert not (1 == 1) or ('1' == '1')
+    assert "abc" and "def" == "def"
```

Comment:
i suppose execution order is not what you expect

it is:
```
"abc" and ("def" == "def")
```

Reply:
fixed